calebwagner/react-notes-app
# added filter notebooks

#### Changes Made
1. Modified file NotebookList.js to include notebooks filtered by user.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cjw-noteslist
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > check NotebookList.js for notebook filter function
    > check in browser for matching user id to notebook id
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
